Metrics Charts Index (#2367)
SumoLogic/sumologic-documentation: docs/metrics/metric-charts/index.md

  <p>Learn how to tailor line charts and area charts for metrics queries.</p>
  </div>
</div>
<div className="box smallbox3 card">
  <div className="container">
  <a href="/docs/metrics/metric-charts/single-value-metric-charts"><img src={useBaseUrl('img/icons/operations/retain-and-visualize-logs.png')} alt="icon" width="40"/><h4>Single Value Charts</h4></a>
  <p>Learn how to create and tailor single value metric charts.</p>
  </div>
</div>
<div className="box smallbox4 card">
  <div className="container">
  <a href="/docs/metrics/metric-charts/interacting-metric-charts"><img src={useBaseUrl('img/icons/operations/human-to-machine.png')} alt="icon" width="40"/><h4>Interact with Metrics Charts</h4></a>
  <p>Learn how to interact with metric charts and use the options on the Query and Legend tabs.</p>
  </div>
</div>
<div className="box smallbox5 card">
  <div className="container">
  <a href="/docs/metrics/metric-charts/quantization-interval-chart"><img src={useBaseUrl('img/icons/observe.png')} alt="icon" width="40"/><h4>Choose a Quantization Interval</h4></a>

>   <p>Learn why its recommended to change the Quantization Interval when running a metric query.</p>

Reviewer: ```suggestion
  <p>Learn why we recommend changing the Quantization Interval when running a metric query.</p>
```
